### ISSUE TITLE
[InfoBox] for Collections

### ISSUE THREAD

**Issue by Podden** [TylerTemp/SaintsField#71]
Is there a way to display ONE Infobox per Array, not ONE Infobox per Array element?
```
    [InfoBox("This is a box")]
    public RuntimePlatform[] platforms;
```
![grafik](https://github.com/user-attachments/assets/639f37d8-5b8b-4723-b1ac-f4b034032303)

**Comment by TylerTemp** [comment]
> Directly using on list/array will apply to every direct element of the list, this is a limit from Unity.

At this version, not yet. We need to add a new `InfoBox` of Playa version, like `PlayaRichLabel` vs `RichLabel` for array/list

**Comment by TylerTemp** [comment]
Hi, this bug has been implemented in [3.3.0](https://github.com/TylerTemp/SaintsField/releases/tag/3.3.0). As it's a breaking changes, please read the changelog before upgrading.

**Comment by TylerTemp** [comment]
This issue is now closed as it's been implemented.

Please comment below if you still have this issue so I can open it again.
